### ISSUE TITLE
CKEditor 5 Link Box hovers underneath the Inline editor Toolbar

### ISSUE THREAD

**Issue by AustinMutschler** [ckeditor/ckeditor5#1757]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

 💻 Version of CKEditor
@ckeditor/ckeditor5-build-inline": "12.1.0"
@ckeditor/ckeditor5-react": "1.1.3"
CKEditor 5

## 📋 Steps to reproduce

1. Have 1 or 2 lines of data in the CKEditor box
2. Highlight any text and click the Link widget

## ✅ Expected result
Link widget input box hovers downward away from the CKEditor inline Header box

## ❎ Actual result
Link widget input box hovers upward underneath the CKEditor inline Header box

Basically, when the input box is too small, CKEditor decides to put the link input popup above the text rather than under it, this causes the input to actually show underneath the CKEditor Inline editor box which means a user cannot click the buttons to edit a link.

**Comment by AustinMutschler** [comment]
This is not isolated to the link widget. Anything that has an overhead popup including tables, displays its options under the CKEditor Header toolbar which causes the widget options to be inaccessible.

**Comment by Mgsy** [comment]
Hello, can you provide some screencast/screenshot and exact steps to reproduce your issue? I'm afraid that I'm not able to reproduce it with given information.

**Comment by AustinMutschler** [comment]
@Mgsy It seems to happen when you have a one or two item list.

![Link Popup under inline editor](https://user-images.githubusercontent.com/6967992/57944912-47182f80-788d-11e9-8a9e-48ffba37fa9d.png)

**Comment by Mgsy** [comment]
I've checked it in the inline build and everything seems to work fine:

![image](https://user-images.githubusercontent.com/20988892/58002386-7bcef580-7ade-11e9-8ad6-868a96605a6a.png)

Can you write more information about your environment? How do you implement CKEditor, are you working with custom styles (maybe some styles overwrite others), did you customize CKEditor?

Additionally, you can check whether your dependencies are updated. You can use [npm-check-updates](https://www.npmjs.com/package/npm-check-updates).

**Comment by AustinMutschler** [comment]
I am using Roboto Regular font. It looks like you have more content below the line. When the list item line is by itself this happens or when there are 2 list items.

**Comment by Mgsy** [comment]
I can confirm the problem. When there is only one block in the editor, the toolbar moves down and it's impossible to use the contextual balloon.

![bug_cke5](https://user-images.githubusercontent.com/20988892/58172944-3f47f900-7c9a-11e9-8223-7f8012435ef4.gif)

**Comment by Reinmar** [comment]
DUP of https://github.com/ckeditor/ckeditor5-ui/issues/460.
